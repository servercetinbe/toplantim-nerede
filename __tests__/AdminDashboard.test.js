import React from 'react';
import { render } from "@testing-library/react";
import AdminDashboard from "../src/app/admin/page";

test("AdminDashboard matches snapshot", () => {
    const { asFragment } = render(<AdminDashboard />);
    expect(asFragment()).toMatchSnapshot();
})