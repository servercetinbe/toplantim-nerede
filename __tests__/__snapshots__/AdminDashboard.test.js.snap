// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDashboard matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1lb6n3v"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-1wxecgq-MuiCircularProgress-root"
      role="progressbar"
      style="width: 60px; height: 60px;"
    >
      <svg
        class="MuiCircularProgress-svg css-54pwck-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-960umw-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="19.5"
          stroke-width="5"
        />
      </svg>
    </span>
  </div>
</DocumentFragment>
`;
